src/OrchardCore.Build/OrchardCore.Commons.props

```diff
@@ -42,6 +43,10 @@
   <ItemGroup>
       <!--This is used for Source Link -->
        <PackageReference Include="Microsoft.SourceLink.GitHub" PrivateAssets="All"/>
+      <None Include="$(MSBuildThisFileDirectory)/../docs/assets/images/nugeticon.png">
+        <Pack>True</Pack>
+        <PackagePath>icon.png</PackagePath>
```

Comment:
Could you show how this ends up in the package after a `dotnet pack`, please?

Reply:
updated desc with image